pom.xml

```diff
@@ -51,7 +51,7 @@
             <dependency>
                 <groupId>software.amazon.awssdk</groupId>
                 <artifactId>bom</artifactId>
-                <version>2.15.47</version>
+                <version>2.20.138</version>
```

Comment:
Don't update this if we don't require it. It can break things.

Reply:
`SdkSystemSetting` for using fips endpoint (`AWS_USE_FIPS_ENDPOINT`) is not available in the sdk versions  that are older than [2.17.79](https://github.com/aws/aws-sdk-java-v2/releases/tag/2.17.79).

So, I've just updated the sdk version to the one we're using in Nucleus.

Reply:
Without this update, some of the integration tests are failing with the following errors for clients in Nucleus: 
```
Can't create instance of com.aws.greengrass.mqttclient.MqttClient
....
Caused by: java.lang.NoSuchFieldError: AWS_USE_FIPS_ENDPOINT
        at com.aws.greengrass.deployment.DeviceConfiguration.lambda$getRegionValidator$3(DeviceConfiguration.java:375)
```

```
Caused by: java.lang.IllegalArgumentException: Can't create instance of com.aws.greengrass.util.GreengrassServiceClientFactory
.....
Caused by: java.lang.NoClassDefFoundError: software/amazon/awssdk/awscore/AwsClient
```